packages/js/test/plugins/auctionHouseModule/createBid.test.ts

```diff
@@ -208,3 +210,158 @@ test('[auctionHouseModule] create public receipt-less bid but cannot fetch it af
+  const nft = await createNft(mx, {}, { owner: seller.publicKey });
+
+  const auctioneerAuthority = Keypair.generate();
+
+  const { client } = await createAuctionHouse(mx, auctioneerAuthority);
+
+  // When we create a private bid on that NFT for 1 SOL.
+  const { bid, buyerTradeState } = await client
+    .bid({
+      mintAccount: nft.mintAddress,
+      seller: seller.publicKey,
+      price: sol(1),
+    })
+    .run();
+
+  // Then we created and returned the new Bid with appropriate defaults.
+  t.equal(bid.tradeStateAddress, buyerTradeState);
+  t.same(bid.price, sol(1));
+  t.same(bid.tokens, token(1));
+  t.false(bid.isPublic);
+});
+
+test('[auctionHouseModule] create public receipt-less Auctioneer bid', async (t: Test) => {
+  // Given we have an Auctioneer Auction House and an NFT.
+  const mx = await metaplex();
+  const seller = await createWallet(mx);
+  const nft = await createNft(mx, {}, { owner: seller.publicKey });
+
+  const auctioneerAuthority = Keypair.generate();
+
+  const { client } = await createAuctionHouse(mx, auctioneerAuthority);
+
+  // When we create a public bid on that NFT for 1 SOL.
+  const { bid, buyerTradeState } = await client
+    .bid({
+      mintAccount: nft.mintAddress,
+      price: sol(1),
+    })
+    .run();
+
+  // Then we created and returned the new Bid with appropriate defaults.
+  t.equal(bid.tradeStateAddress, buyerTradeState);
+  t.same(bid.price, sol(1));
+  t.same(bid.tokens, token(1));
+  t.ok(bid.isPublic);
+});
+
+test('[auctionHouseModule] it throws an error if Buy is not included in Auctioneer scopes', async (t: Test) => {
+  // Given we have an NFT.
+  const mx = await metaplex();
+  const seller = await createWallet(mx);
+  const nft = await createNft(mx, {}, { owner: seller.publicKey });
+
+  const auctioneerAuthority = Keypair.generate();
+
+  // Create Auctioneer Auction House to only allow Sell.
+  const { client } = await createAuctionHouse(mx, auctioneerAuthority, {
+    auctioneerScopes: [AuthorityScope.Sell],
+  });
+
+  // When we create a private bid on that NFT for 1 SOL.
+  const promise = client
+    .bid({
+      mintAccount: nft.mintAddress,
+      seller: seller.publicKey,
+      price: sol(1),
+    })
+    .run();
+
+  // Then we expect an error.
+  await assertThrows(
+    t,
+    promise,
+    /The Auctioneer does not have the correct scope for this action/
+  );
+});
+
+test('[auctionHouseModule] it allows to Buy after Auctioneer scope update', async (t: Test) => {
+  // Given we have an NFT.
+  const mx = await metaplex();
+  const seller = await createWallet(mx);
+  const nft = await createNft(mx, {}, { owner: seller.publicKey });
+
+  const auctioneerAuthority = Keypair.generate();
+
+  // Create Auctioneer Auction House to only allow Sell.
+  const { auctionHouse, client } = await createAuctionHouse(
+    mx,
+    auctioneerAuthority,
+    {
+      auctioneerScopes: [AuthorityScope.Sell],
+    }
+  );
+
+  // When we update scope to allow Buy.
```

Comment:
Great test! Just updating some of the comments slightly.

```suggestion
  // And an Auctioneer Auction House that, at first, could only Sell.
  const { auctionHouse, client } = await createAuctionHouse(
    mx,
    auctioneerAuthority,
    { auctioneerScopes: [AuthorityScope.Sell] },
  );

  // But was later on updated to also allow the Buy scope.
```